chromium_src/chrome/browser/net/proxy_config_monitor.cc

```diff
@@ -33,3 +35,6 @@ std::unique_ptr<net::ProxyConfigService> CreateProxyConfigServiceTor(
 #endif
 
 #include "../../../../../chrome/browser/net/proxy_config_monitor.cc"  // NOLINT
+
+// Required by lint
+#include "brave/chromium_src/chrome/browser/net/proxy_config_monitor.h"
```

Comment:
this should go at the top, right?

Reply:
can we actually disable this lint check for the chromium_src dir in chromium_src/CPPLINT.cfg?

Reply:
fixed in 3bc4178c9ee409e6126e0f5055bd86afe5f687e9